ic-cdk/src/api.rs

```diff
@@ -560,6 +564,53 @@ pub fn cost_vetkd_derive_key<T: AsRef<str>>(
     sign_cost_result(cost, code)
 }
 
+/// Gets the number of environment variables available in the canister.
+pub fn env_var_count() -> usize {
+    ic0::env_var_count()
+}
+
+/// Gets the size of the name of the environment variable at the given index.
+///
+/// # Panics
+///
+/// This function traps if the index is out of bounds (>= than value provided by [`env_var_count`])
+pub fn env_var_name(index: usize) -> String {
+    let len = ic0::env_var_name_size(index);
+    let mut buf = vec![0u8; len];
+    ic0::env_var_name_copy(index, &mut buf, 0);
+    String::from_utf8_lossy(&buf).into_owned()
```

Comment:
We should panic on an unexpected value instead of garbling it. It converts faster too.